cmd/boulder-wfe2/main.go

```diff
@@ -50,16 +51,34 @@ type Config struct {
 
 		RAService *cmd.GRPCClientConfig
 		SAService *cmd.GRPCClientConfig
+
 		// GetNonceService contains a gRPC config for any nonce-service instances
 		// which we want to retrieve nonces from. In a multi-DC deployment this
-		// should refer to any local nonce-service instances.
+		// should refer to local nonce-service instances only.
 		GetNonceService *cmd.GRPCClientConfig
+
 		// RedeemNonceServices contains a map of nonce-service prefixes to
 		// gRPC configs we want to use to redeem nonces. In a multi-DC deployment
 		// this should contain all nonce-services from all DCs as we want to be
 		// able to redeem nonces generated at any DC.
+		//
+		// DEPRECATED: See RedeemNonceService, above.
+		// TODO (#6610) Remove this after all configs have migrated to
+		// `RedeemNonceService`.
 		RedeemNonceServices map[string]cmd.GRPCClientConfig
 
+		// RedeemNonceService contains a gRPC config for each nonce-service
+		// instance which we want to redeem nonces at. In a multi-DC deployment
+		// this should contain both local and remote nonce-service instances.
```

Comment:
This comment ("contains a gRPC config for each...") sounds like RedeemNonceService will be a slice of configs, rather than being a single gRPC config with a slice of SRVLookups.

Reply:
Good call. I've also given `GetNonceService` an update to keep things consistent.